runners/google-cloud-dataflow-java/examples/build.gradle

```diff
@@ -19,7 +19,7 @@
 import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(publish: false, exportJavadoc: false)
+applyJavaNature(publish: false, exportJavadoc: false, setAutomaticModuleName: false)
```

Comment:
I suppose this does not have automtatic module name because people are not suppose to build from it? We should probably document this as part of `setAutomaticModuleName`in `BeamModulePlugin.groovy`. In what particular reasons we do not set automatic module names. I suppose this will be the case also for some test pure modules like the Elasticsearch ones.

Reply:
I'm still not 100% sure if we should leave modules without these headers, to be honest. My motivation was to skip it in aggregated docs (javadoc module), examples and testing modules because we know that people should not depend on it (do we?). I also find it as a form of discouragement to depend on them so but a very weak one.